### PR TITLE
Display comments on main page (index.html) and restore random TV quot…

### PR DESCRIPTION
**Pull request by ryan-draper**
…e generator.

Screenshot 2020-06-06 at 1 14 06 AM

I have also re-deployed my site, so you can check out these changes at http://ryandraper-step-2020.appspot.com/.

### CODE REVIEW

**Review by charu0291 — APPROVED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -29,9 +33,6 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```
Comment (curves4d) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: update comment

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -117,7 +117,7 @@ <h3>What I'm Listening To</h3>
           <h3>TV Quotes</h3>
           <div id="button-section">
             <p>Click here to get a quote from some of Ryan's favorite TV shows:</p>
-            <button onclick="getQuoteFromServer()">here</button>
```
Comment (curves4d) [inline portfolio/src/main/webapp/index.html:120]: Usually it's better practice to keep all js code inside the js file. You can add these handlers inside a call to window.addEventListener('load', func() { ... } );

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -13,7 +13,7 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
   </head>
-  <body>
+  <body onload="loadComments()">
```
Comment (curves4d) [inline portfolio/src/main/webapp/index.html:16]: Same as the comment below

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (ryan-draper) [reply portfolio/src/main/webapp/index.html:120]: All of my js code is inside of the js file. Also, the code you commented on got removed. getQuoteFromServer() (which got replaced with loadComments()) calls a function in my js file. I'm not sure how window.addEventListener works, but it probably does essentially the same thing as calling these js functions, right?

Comment (ryan-draper) [reply portfolio/src/main/webapp/index.html:120]: I did more research into event listeners, and I think I understand the basic idea. We learned from the tutorials to call the methods as I did above, but if you think changing it to use event listeners is worthwhile I am happy to do so.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (ryan-draper) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Done.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (ryan-draper) [reply portfolio/src/main/webapp/index.html:120]: I will address these changes in a new branch.

**Review by curves4d — APPROVED** [review]

**Pull request merged**